src/main/java/seedu/address/logic/commands/GetPrev3MonthFeeCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+/**
+ * Gets current month and previous 2 months expected monthly fee based on the current list of sessions
+ */
+public class GetPrev3MonthFeeCommand extends Command {
+    public static final String COMMAND_WORD = "3Month_Fee";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows the total expected monthly fee"
+        + "for this month + previous 2 months";
```

Comment:
"Shows the totalled fee per month, for the past 3 months." Or something of the sort could be somewhat more accurate.

Reply:
Alright! Change from that to "Shows the totalled fees per month, for this month + previous 2 months" instead.